ui/apps/platform/cypress.config.js

```diff
@@ -25,7 +24,6 @@ module.exports = {
 
     e2e: {
         baseUrl: 'https://localhost:3000',
-        specPattern: 'cypress/integration/**/*.test.{js,ts}',
```

Comment:
For clarity, this is removed and specified explicitly in the `cypress.sh` and `cypress-ocp.sh` startup scripts.